packages/shared/src/components/ShareBar.tsx

```diff
@@ -42,53 +44,70 @@ export default function ShareBar({ post }: { post: Post }): ReactElement {
     onClick(ShareProvider.CopyLink);
   };
 
+  const onShareToSquad = (squad: Squad) => {
+    trackEvent(postAnalyticsEvent(AnalyticsEvent.StartShareToSquad, post));
+    openModal({
+      type: LazyModal.CreateSharedPost,
+      props: {
+        squad,
+        preview: post,
+        onSharedSuccessfully: () => {
+          return trackEvent(
+            postAnalyticsEvent(AnalyticsEvent.ShareToSquad, post),
+          );
```

Comment:
This would do right?
```suggestion
        onSharedSuccessfully: () => trackEvent(
            postAnalyticsEvent(AnalyticsEvent.ShareToSquad, post),
          )
```